internal/commands/app/create_inputs.go

```diff
@@ -82,9 +82,9 @@ func (i *createInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 	return nil
 }
 
-func (i *createInputs) resolveName(ui terminal.UI, client realm.Client, r appRemote) error {
+func (i *createInputs) resolveName(ui terminal.UI, client realm.Client, filter realm.AppFilter) error {
```

Comment:
Changed this because the FindApps methods was looking for **client app id** while we were trying to find a match for an **app id**